atom/browser/api/lib/auto-updater/auto-updater-win.js

```diff
@@ -2,70 +2,62 @@ const app = require('electron').app;
 const EventEmitter = require('events').EventEmitter;
 const url = require('url');
 const squirrelUpdate = require('./squirrel-update-win');
+const util = require('util');
 
-var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-hasProp = {}.hasOwnProperty;
+function AutoUpdater() {
+  AutoUpdater.super_.call(this);
```

Comment:
That's true. The main benefit of calling the `EventEmitter` explicitly is that anyone reading the constructor code will know that it extends `EventEmitter`. Otherwise you need to search for the `util.inherits` call to see what you are extending.
The better option is to use es2015 classes (e.g. `class AutoUpdater extends EventEmitter { ... }`).